EKF/ekf_helper.cpp

```diff
@@ -1623,17 +1622,17 @@ void Ekf::calcExtVisRotMat()
 
 	// convert filtered vector to a quaternion and then to a rotation matrix
 	q_error.from_axis_angle(_ev_rot_vec_filt);
-	_ev_rot_mat = quat_to_invrotmat(q_error);
+	q_error.normalize();
+	_ev_rot_mat = Dcmf(q_error); // rotation from EV reference to EKF reference
 
 }
 
 // reset the estimated misalignment between the EV navigation frame and the EKF navigation frame
 // and update the rotation matrix which rotates EV measurements into the EKF's navigation frame
 void Ekf::resetExtVisRotMat()
 {
-	// calculate the quaternion delta between the EKF and EV reference frames at the EKF fusion time horizon
-	Quatf quat_inv = _ev_sample_delayed.quat.inversed();
-	Quatf q_error =   quat_inv * _state.quat_nominal;
+	// calculate the quaternion delta between the EV and EKF reference frames at the EKF fusion time horizon
```

Comment:
// calculate the quaternion delta from the EV to the EKF reference frame at the EKF fusion time horizon